azure-pipelines.yml

```diff
@@ -88,6 +88,7 @@ jobs:
     - task: VSBuild@1
       inputs:
         solution: 'rpcs3.sln'
+        vsVersion: 16.6
```

Comment:
maybe try
```suggestion
        vsVersion: '16.6'
```

Reply:
GalCiv already tried and that was not correct. Documentation has it unquoted and only contains major versions, so the correct one would actually be `16.0`. However, Azure automagically accounted for this, and it still failed so idk

Relevant line in the build log:
> ##[warning]Visual Studio version '16.6' not found. Falling back to version '16.0'.

Reply:
try 15 then xd

Reply:
I already tried that, it doesn't work.

Reply:
> try 15 then xd

actually trying it, it may just work

Reply:
fails with

> ##[warning]Visual Studio version '15.0' not found. Falling back to version '16.0'.

Absolutely cursed and 15.x wouldn't work well anyways. I guess rip our Windows builds for a while?